### ISSUE TITLE
Update `suggest-stacks` command to point to Paketo images instead of Cloudfoundry

### ISSUE THREAD

**Issue by jromero** [buildpacks/pack#552]
> Oh I didn't even realize there's a `suggest-stacks` command 😄 . Yeah, we will need to make this update for stacks at some point, but I think that we should wait on this for now. Reason being, we're planning on adding mixins in pretty soon so that we can replace cflinuxfs3 with the bionic stack + mixins. In the interim, stack wise, we'll be publishing build/run stack images to the following paths:
> 
> Base: `paketobuildpacks/[build/run]:base-cnb`
> Tiny: `paketobuildpacks/[build/run]:tiny-cnb`
> **Full**: `paketobuildpacks/[build/run]:full-cnb-cf`
> 
> Note that we're publishing **full** to a temporary path to avoid breaking stuff when we add mixin support. Once that's added the path for the Full stack becomes `paketobuildpacks/[build/run]:full-cnb` and we can get this command updated!

_Originally posted by @kvedurmu in https://github.com/buildpacks/pack/issues/551#issuecomment-606277848_

**Comment by jromero** [comment]
@kvedurmu A few questions to make this actionable:

This is the output right now:

```
pack suggest-stacks

Stacks maintained by the community:

    Stack ID: heroku-18
    Description: The official Heroku stack based on Ubuntu 18.04
    Maintainer: Heroku
    Build Image: heroku/pack:18-build
    Run Image: heroku/pack:18

    Stack ID: io.buildpacks.stacks.bionic
    Description: A minimal Cloud Foundry stack based on Ubuntu 18.04
    Maintainer: Cloud Foundry
    Build Image: cloudfoundry/build:base-cnb
    Run Image: cloudfoundry/run:base-cnb

    Stack ID: org.cloudfoundry.stacks.cflinuxfs3
    Description: A large Cloud Foundry stack based on Ubuntu 18.04
    Maintainer: Cloud Foundry
    Build Image: cloudfoundry/build:full-cnb
    Run Image: cloudfoundry/run:full-cnb

    Stack ID: org.cloudfoundry.stacks.tiny
    Description: A tiny Cloud Foundry stack based on Ubuntu 18.04, similar to distroless
    Maintainer: Cloud Foundry
    Build Image: cloudfoundry/build:tiny-cnb
    Run Image: cloudfoundry/run:tiny-cnb
```

1. Is it worth only suggesting `base-cnb` and `tiny-cnb` given `full-cnb-cf` could be a work in progress? We can always add it later.
2. Do you know what the stack ID for `tiny-cnb` will be?

If you or someone from the Paketo team could provide the information in the format it will be displayed (or even better put up a PR) we can get this in pretty fast.

**Comment by kvedurmu** [comment]
@jromero IMO this request is fairly low priority and we should wait on implementing this feature until our mixins work is complete. This seems like a fairly simple change, so we'd be happy to PR these changes in at that point. 

@martys34 @sclevine could one of you answer the second question? 

Also one note from the original post, we've decided to publish all Paketo artifacts to this GCR path: `gcr.io/paketo-buildpacks`. So for this change the paths will look like:
Base: `gcr.io/paketo-buildpacks/[build/run]:base-cnb`
Tiny: `gcr.io/paketo-buildpacks/[build/run]:tiny-cnb`
Full: `gcr.io/paketo-buildpacks/[build/run]:full-cnb`

**Comment by jromero** [comment]
@kvedurmu that seems fair my only concern is with 0.10.0, we would be suggesting cloud foundry stacks. As long as your team is ok with that and those stacks stay functional then I would agree with pushing back til a future release.

**Comment by dfreilich** [comment]
Closed in #559
